addons/l10n_cl/views/report_invoice.xml

```diff
@@ -229,4 +250,69 @@
+    <template id="tax_totals_widget" inherit_id="account.document_tax_totals">
+        <t t-foreach="tax_totals['subtotals']" t-as="subtotal" position="replace">
+            <t t-if="o.country_code != 'CL'">
+                <t t-foreach="tax_totals['subtotals']" t-as="subtotal">
+                    <tr class="border-black o_subtotal">
+                        <td><strong t-esc="subtotal['name']"/></td>
+
+                        <td class="text-end">
+                            <span t-att-class="oe_subtotal_footer_separator" t-esc="subtotal['formatted_amount']"/>
+                        </td>
+                    </tr>
+
+                    <t t-set="subtotal_to_show" t-value="subtotal['name']"/>
+                    <t t-call="account.tax_groups_totals"/>
+                </t>
+            </t>
+            <t t-else="">
+                <t t-set="subtotal_amounts" t-value="o._l10n_cl_get_amounts()"/>
+                <t t-set="withholdings" t-value="o._l10n_cl_get_withholdings()"/>
+                <t t-if="subtotal_amounts['subtotal_amount_taxable'] != 0.0">
+                    <tr class="border-black is-subtotal"><td><strong>Amount Untaxed</strong></td><td class="text-end oe_subtotal_footer_separator" t-esc="subtotal_amounts['subtotal_amount_taxable']" t-options="{'widget': 'monetary', 'display_currency': subtotal_amounts['main_currency']}"/></tr>
+                </t>
+                <t t-if="subtotal_amounts['subtotal_amount_exempt'] != 0.0">
+                    <tr class="border-black is-subtotal">
+                        <td><strong>Amount Exempt</strong></td>
+                        <td class="text-end oe_subtotal_footer_separator" t-esc="subtotal_amounts['subtotal_amount_exempt']"
+                            t-options="{'widget': 'monetary', 'display_currency': subtotal_amounts['main_currency']}"/>
+                    </tr>
+                </t>
+                <t t-if="subtotal_amounts['vat_amount'] != 0.0">
+                    <tr>
+                        <td t-esc="'IVA %s' % subtotal_amounts['vat_percent']"/>
+                        <td class="text-end" t-esc="subtotal_amounts['vat_amount']"
+                            t-options="{'widget': 'monetary', 'display_currency': subtotal_amounts['main_currency']}"/></tr>
+                </t>
+                <t t-foreach="withholdings" t-as="wh">
+                    <tr>
+                        <td t-esc="'Ret %s sobre %s' % (wh['tax_name'], o._format_lang_totals(wh['tax_base'], subtotal_amounts['main_currency']))"/>
+                        <td class="text-end" t-esc="wh['tax_amount']" t-options="{'widget': 'monetary', 'display_currency': subtotal_amounts['main_currency']}"/></tr>
+                </t>
+            </t>
+        </t>
+        <tr class="border-black o_total" position="replace">
+            <tr class="border-black o_total" position="replace">
+                <t t-if="o.country_code != 'CL'">
+                    <tr class="border-black o_total">
```

Comment:
$0 ?

Reply:
removed since instead of that we don't use $0 because theoretically with don't need to check for country code here 254506c926